### PR TITLE
Fix/persistences

### PR DESCRIPTION
**Pull request by Trjegul84**

### CODE REVIEW

File: services/connectivity/src/persistence/connectivity.js [hunk 1]
```diff
@@ -20,16 +20,13 @@ module.exports = (
    * @returns {Object[]} Values of area distribution for each category of protected area
    * connectivity
    */
-  findCurrentPAConnectivity: async (areaType, areaId, year = 2020) => {
-    try {
-      return connectivity.query()
-        .where({ geofence_type: areaType, geofence_id: areaId, prot_year: year })
-        .select('protconn', 'protunconn', 'unprotected', 'area_ha');
-    } catch (e) {
+  findCurrentPAConnectivity: async (areaType, areaId, year = 2020) => connectivity.query()
```
Comment (erikasv) [inline services/connectivity/src/persistence/connectivity.js:23]: Put the function body in another line, see these [examples](https://github.com/airbnb/javascript#arrows--one-arg-parens) (the good sections)

Comment (erikasv) [reply services/connectivity/src/persistence/connectivity.js:23]: Apply this for all functions

Comment (erikasv) [reply services/connectivity/src/persistence/connectivity.js:23]: Also you should delete the `async`

Comment (erikasv) [reply services/connectivity/src/persistence/connectivity.js:23]: see `findTotalNumberOfSpecies` in `richness_nos.js` as reference

File: services/diversity/src/persistence/richness_nos.js [hunk 2]
```diff
@@ -19,22 +19,20 @@ module.exports = (
...
-          db.raw('coalesce(rnr.total_inf, 0) as region_inferred'),
-        )
-        .leftJoin('richness_nos_regions as rnr', 'rn.id_region', 'rnr.id_region')
-        .where({ 'rn.geofence_type': areaTypeKeys(areaType), 'rn.geofence_id': areaId });
-    } catch (e) {
-      logger.error(e.stack || e.Error || e.message || e);
-      throw new Error('Error getting data');
-    }
-  },
+  findTotalNumberOfSpecies: async (areaType, areaId) => (
```
Comment (erikasv) [inline services/diversity/src/persistence/richness_nos.js:22]: you should delete de `async` here too

File: services/diversity/src/persistence/richness_nos.js [hunk 3]
```diff
@@ -44,22 +42,19 @@ module.exports = (
...
-        .select(
-          db.raw('coalesce(rn.end_inf, 0) as inferred'),
-          db.raw('coalesce(rn.end_obs, 0) as observed'),
-          db.raw('coalesce(rnr.end_obs, 0) as region_observed'),
-          db.raw('coalesce(rnr.end_inf, 0) as region_inferred'),
-        )
-        .leftJoin('richness_nos_regions as rnr', 'rn.id_region', 'rnr.id_region')
-        .where({ 'rn.geofence_type': areaTypeKeys(areaType), 'rn.geofence_id': areaId });
-    } catch (e) {
+  findEndemicNumberOfSpecies: async (areaType, areaId) => db('richness_nos as rn')
```
Comment (erikasv) [inline services/diversity/src/persistence/richness_nos.js:45]: Add new line for function body

File: services/forest/src/persistence/SCIHF.js [hunk 4]
```diff
@@ -12,19 +12,16 @@ module.exports = (
...
-  findSCIHFInEA: async (areaId, year = 2018) => {
-    try {
-      return geoIntegrity.query()
-        .where({ id_ea: areaId, sci_year: year })
-        .select('hf_pers', 'sci_cat', 'binary_protected')
-        .sum('area_ha as area')
-        .groupBy('binary_protected', 'sci_cat', 'hf_pers')
-        .orderBy('binary_protected', 'sci_cat', 'hf_pers');
-    } catch (e) {
+  findSCIHFInEA: async (areaId, year = 2018) => geoIntegrity.query()
```
Comment (erikasv) [inline services/forest/src/persistence/SCIHF.js:15]: Same with the function body

**Review by erikasv — CHANGES_REQUESTED** [review]

File: services/diversity/src/persistence/richness_nos.js (hunk 2 above)
Comment (Trjegul84) [reply services/diversity/src/persistence/richness_nos.js:22]: Should I delete async in findSCIHFLayerInEA of SCIHF,js ? as in the richness_nos we do not use async.

Comment (Trjegul84) [reply services/diversity/src/persistence/richness_nos.js:22]: also richness_nos neither use try and catch for the raw querys. Is it correct to remove try-catch raw ?

**Review by erikasv — APPROVED** [review]

**Pull request merged**